### PR TITLE
Update WR (gl trait, scroll roots)

### PR DESCRIPTION
**Pull request by glennw**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/16073)

### CODE REVIEW

File: components/compositing/compositor.rs [hunk 1]
```diff
@@ -291,34 +293,34 @@ impl RenderTargetInfo {
     }
 }
 
-fn initialize_png(width: usize, height: usize) -> RenderTargetInfo {
-    let framebuffer_ids = gl::gen_framebuffers(1);
-    gl::bind_framebuffer(gl::FRAMEBUFFER, framebuffer_ids[0]);
+fn initialize_png(gl: &Rc<gl::Gl>, width: usize, height: usize) -> RenderTargetInfo {
```
Comment (emilio) [inline components/compositing/compositor.rs]: nit: Probably this only needs `&gl::Gl` instead of the whole `Rc`.

**Review by emilio — APPROVED** [review]
r=me % nit

**Comment by jdm** [comment]
```
  ▶ PASS [expected FAIL] /css-transforms-1_dev/html/transform-3d-rotateY-stair-above-001.htm

  ▶ PASS [expected FAIL] /css-transforms-1_dev/html/css-rotate-2d-3d-001.htm

  ▶ FAIL [expected PASS] /css-transforms-1_dev/html/regions-transforms-005.htm
  └   → /css-transforms-1_dev/html/regions-transforms-005.htm 1633d01afdfc1ea2419c317355187409cde90302
/css-transforms-1_dev/html/reference/regions-transforms-001-ref.htm 8f444b88a6bfe9bac67ada3454443e89415231b3
Testing 1633d01afdfc1ea2419c317355187409cde90302 == 8f444b88a6bfe9bac67ada3454443e89415231b3
```

**Comment by glennw** [comment]
I was expecting those two new passes. I'll see if I can repro that new failure locally.

**Comment by glennw** [comment]
Oh, the ```regions-transforms-005.htm``` failure is already an expected failure on Linux - now it's just consistent between platforms. This is likely due to the improvements in pixel snapping with transforms in this update - but I'll confirm that.

**Comment by glennw** [comment]
@bors-servo retry
 * Test if those timeouts are consistent.

**Comment by glennw** [comment]
@bors-servo try

I ran the tests locally with the WR update and without it. The timings were approximately the same, perhaps a little faster with the WR update. So let's try again with updated tests and see if the timeouts occur again.

**Comment by glennw** [comment]
Try run succeeded - the code changes have been signed off by @emilio - so justs needs someone to sign off on the test expectation changes.

**Pull request merged**